Lesson33/Gokhan/src/components/Account/index.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { useUserContext } from '../Context/UserContext';
+import './styles.css';
+
+export const Account = () => {
+  const { state, dispatch } = useUserContext();
```

Comment:
You don't need context in this component, the data is used in UserDetailsForm, import it there instead